source/Generic/ExtraMetadataToolsVideo/ExtraMetadataToolsVideo.psm1

```diff
@@ -967,9 +1183,10 @@ function Invoke-YoutubeSearchWindow
                         <Image Margin="3" Height="80" Source="{Binding Thumbnail}"/>
                         <StackPanel Margin="5,0,0,0">
                             <TextBlock Margin="3" Text="{Binding Name}"/>
+                            <TextBlock Margin="0" Text="{Binding ChannelName}"/>
```

Comment:
Here it should have the same margin or at least left margin as the other textblocks for alignment.

I think all left margins should be removed and just be set from the stack panel to `8`, only keeping the margins for below, like `Margin="0,0,0,3"`

Reply:
I hope i understood you right there. Just let me know if that was wrong.
Done in cb8701e19ceef61080e5c66542ce44b05b026f96